tests/cpu_bugs/meltdown.pm

```diff
@@ -77,12 +78,22 @@ sub check_cpu_flags {
         }
     }
 }
-
 sub run {
+    my ($self) = shift;
     select_console 'root-console';
     my $obj = meltdown->new($mitigations_list);
     #run base function testing
     $obj->do_test();
+    #extra testing for nopti
+    add_grub_cmdline_settings("nopti");
```

Comment:
taking into account #8999  I would say that it is plus one reason to extract this into a function :)

Reply:
Thanks, already changed